### PR TITLE
Feat: WB-2033, add ColorPickerItem component

### PR DESCRIPTION
**Pull request by jcbe-ode**
# Description

* Ajout du composant ColorPickerItem
* Intégration de la Dropdown refacto
* Refonte de la Toolbar
* Le tout pour permettre de développer le composant TableToolbar pour l'éditeur riche (voir playground)

## Which Package changed?

Please check the name of the package you changed

- [X] Components
- [ ] Core
- [X] Icons
- [X] Hooks

## Has the documentation changed?

- [X] Storybook

## Type of change

Please check options that are relevant.

- [ ] Chore (PATCH)
- [ ] Doc (PATCH)
- [ ] Bug fix (PATCH)
- [X] New feature (MINOR)
- [ ] Breaking change (MAJOR)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: packages/react/src/components/Toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -9,76 +9,56 @@ import {
...
+import { Button, ButtonProps, IconButton, IconButtonProps } from "../Button";
+import { Dropdown, DropdownProps } from "../Dropdown";
+
+interface Item {
+  /** Object type */
+  type: string;
+  /** Items should be named, except for dividers */
+  name?: string;
+  /** ShoHidew item when truthy */
+  isHidden?: boolean;
```
Comment (ccreusat) [inline packages/react/src/components/Toolbar/Toolbar.tsx:23]: @jcbe-ode `isHidden` à renommer `visibility` et peut recevoir `show | hide` comme vu ensemble sur slack 👍🏼

File: packages/react/src/components/Toolbar/Toolbar.tsx [hunk 2]
```diff
@@ -196,56 +175,62 @@ const Toolbar = forwardRef(
...
-                aria-label={t(item.label)}
-                variant="filled"
-                color="primary"
-                tabIndex={index === 0 ? 0 : -1}
-                onKeyDown={handleKeyDown}
-                disabled={isDisabled}
-              />
-            );
+        {items.map((item, index) => {
+          if (item.isHidden === false) return null;
```
Comment (ccreusat) [inline packages/react/src/components/Toolbar/Toolbar.tsx:179]: `if (item.visibility === 'hide' ou !== de 'show') return null`

**Review by david-cc — APPROVED** [review]

**Pull request merged**